indexd/index/drivers/alchemy.py

```diff
@@ -1175,9 +1180,16 @@ def migrate_10(session, **kwargs):
         .format(tb=IndexRecord.__tablename__))
 
 
+def migrate_11(session, **kwargs):
+    session.execute(
+        "ALTER TABLE {} ADD COLUMN rbac VARCHAR;"
+        .format(IndexRecord.__tablename__))
```

Comment:
I was thinking to add this here:

```python
    for record in session.query(IndexRecord).options(joinedload(IndexRecord.acl)):
        if '*' in record.acl:
            record.rbac = '*'
        else:
            record.rbac = ' or '.join(record.acl)
```

But reading more about the user-sync logic, I feel like it might be just not as easy as above. So I'll create a new ticket to address that.